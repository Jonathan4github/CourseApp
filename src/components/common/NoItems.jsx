import React from 'react';
import PropTypes from 'prop-types';


const propTypes = {
  displayText: PropTypes.string.isRequired
};

const NoItems = ({ displayText }) => (
  <div className="no-items">
    <i className="fa fa-exclamation-triangle fa-3x pb-3 d-block" />
    <p className="lead">{displayText}</p>
  </div>
);

NoItems.propTypes = propTypes;

export default NoItems;
