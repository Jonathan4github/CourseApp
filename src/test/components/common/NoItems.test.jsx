/* eslint-disable */
import NoItems from '../../../components/common/NoItems';

const props = {
  displayText: ''
};
describe('<NoItems /> component test', () => {
  it('renders component without crashing', () => {
    const shallowWrapper = shallow(<NoItems {...props} />);
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
