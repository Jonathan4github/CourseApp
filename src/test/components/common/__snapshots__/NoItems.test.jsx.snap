// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoItems /> component test renders component without crashing 1`] = `
<div
  className="no-items"
>
  <i
    className="fa fa-exclamation-triangle fa-3x pb-3 d-block"
  />
  <p
    className="lead"
  />
</div>
`;
